src/components/productCard/ProductCard.jsx

```diff
@@ -1,18 +1,77 @@
-//import { Link } from "react-router-dom";
+import { Link } from "react-router-dom";
+import { useData } from "../../contexts";
+import {
+  getWishlist,
+  getCartlist,
+  addCartlist,
+  editCartlist,
+  addWishlist,
+  deleteWishlist,
+} from "../../services";
+
 const ProductCard = ({ product }) => {
+  const { data, dispatch, token } = useData();
+
+  const wish = data.wishlist.some((item) => item._id === product._id)
+    ? "fas fa-heart wishlisted"
+    : "far fa-heart";
+
```

Comment:
Suggestion:
You can add svgs so that UI is not dependent on websites for icons

Reply:
ok, Thank you for the suggestion